boards/arm/stm32h7/nucleo-h743zi/Kconfig

```diff
@@ -27,4 +27,66 @@ config STM32_ROMFS_IMAGEFILE
         depends on STM32_ROMFS
         default "../../../rom.img"
 
+
+config PROGMEM_OTA_PARTITION
+	bool "Progmem ota partitions"
+	default y
+	select MTD
+	select MTD_BYTE_WRITE
+	select MTD_PARTITION
+	select MTD_PROGMEM
+	select MTD_PROGMEM_ERASESTATE
+
+menuconfig APP_FORMAT_MCUBOOT
+	bool "MCUboot-bootable format"
+	default y
+	select PROGMEM_OTA_PARTITION
+	---help---
+		The MCUboot support of loading the firmware images.
+
+if APP_FORMAT_MCUBOOT
+
+config MCUBOOT_BOOTLOADER
+	bool "MCUboot bootloader application"
+	default y
+	---help---
+		This switch between linker scripts to allow an application be
+		built to another entry point address.
```

Comment:
https://github.com/pkarashchenko/incubator-nuttx-apps/blob/master/boot/mcuboot/Kconfig has same configuration option
```
config MCUBOOT_BOOTLOADER
	bool "MCUboot bootloader application"
	default n
	select BOARDCTL
	select BOARDCTL_BOOT_IMAGE
	---help---
		MCUboot bootloader application
```
I'm not sure if it will work with two configuration options with the same name

Reply:
Yeah you are right. It does not seem to fail but I am gonna change the name to be cleaner. I would use `CONFIG_USE_MCUBOOT_BOOTLOADER`. 

Any suggestion?

Reply:
I would suggest `CONFIG_APP_MCUBOOT_BOOTLOADER`